katsdpcontroller/generator.py

```diff
@@ -397,11 +405,34 @@ def _make_fgpu(
                default=stream.n_samples_between_spectra, initial_status=Sensor.Status.NOMINAL),
         Sensor(int, f"{stream.name}-spectra-per-heap",
                "Number of spectrum chunks per heap",
-               default=stream.n_spectra_per_heap, initial_status=Sensor.Status.NOMINAL)
+               default=stream.n_spectra_per_heap, initial_status=Sensor.Status.NOMINAL),
+        # Note: reports baseband centre frequency, not on-sky, and hence not
+        # stream.centre_frequency.
+        Sensor(float, f"{stream.name}-center-freq",
```

Comment:
Did you deliberately spell "center" here differently?

Reply:
Yes. Because that's how it was spelt by CBF in MeerKAT. Whereas CAM spelt it "center". :man_shrugging: